app/routers/file_uploader.py

```
@@ -0,0 +1,47 @@
+
+    # Create fresh clients/buckets each call (so the patch will work):
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(BUCKET_NAME)
+
+    try:
+        blob = bucket.blob(gcs_file_path)
+        blob.upload_from_file(file.file, content_type="application/pdf")
+
+    except Exception as e:
+        logging.error("Error Traceback: %s", traceback.format_exc())
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)